### PR TITLE
feat: dynamic forms

### PR DESCRIPTION
**Pull request by AysKrimn**
When user create kucuk kucuk buluntu form in admin panel, we create dynamic form for new kucuk buluntu.

### CODE REVIEW

**Comment by AysKrimn** [comment]
i'll improve it later. I finally figured out to capture dynamic inputs.

File: archaeologyMain/apps/buluntuForm/forms.py [hunk 1]
```diff
@@ -26,28 +32,77 @@ class Meta:
...
+
+        # multiple alanlar için
+        for form_class, related_instance in self.form_classes_instances:
+            if related_instance.count() > 0:
+                instance = related_instance.first()
+                form_instance = form_class(instance=instance)
+                for field_name, field_value in form_instance.fields.items():
+                    # falsy değerleri ortadan kaldir
+                    if getattr(instance, field_name) is not None:
+                        self.fields[field_name] = form_instance.fields[field_name]
```
Comment (CoderMungan) [inline archaeologyMain/apps/buluntuForm/forms.py:62]: good!

File: archaeologyMain/apps/buluntuForm/forms.py [hunk 2]
```diff
@@ -26,28 +32,77 @@ class Meta:
...
+        else:
+
+            print("ERRR:", custom_form.errors)
+
+        """
+        for field_name, field in self.fields.items():
+            print("INSTANCE FIELD:", field_name)
+            if field_name in post_data:
+                # Use the field_name to set a new value
+                self.fields[field_name].initial = post_data[field_name]
```
Comment (CoderMungan) [inline archaeologyMain/apps/buluntuForm/forms.py:101]: good!

**Review by CoderMungan — APPROVED** [review]
LGTM!

**Pull request merged**